.buildkite/format_tap.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
```

Comment:
hum... can we use the perl version of this that's in the latest sytest images instead?